.ado/templates/apple-ensure-valid-cocoapods.yml

```diff
@@ -0,0 +1,10 @@
+# Cocoapods 1.15.0 doesn't work with React Native
+steps:
```

Comment:
Where do we call this template?

Reply:
In a file that I forgot to add to this PR. 🤦